scripts/coderabbit_retry/coderabbit_retry.py

```
@@ -0,0 +1,324 @@
+        repo_name: Repository in ``owner/repo`` format.
+
+    Returns:
+        List of matching Issue objects (PRs), capped at ``MAX_PRS``.
+    """
+    since_date = (datetime.now(tz=UTC) - timedelta(days=SINCE_DAYS)).strftime(format="%Y-%m-%d")
+    query = (
+        f"repo:{repo_name} is:pr is:open draft:false "
+        f"-label:Stale -label:wip -label:has-conflicts "
+        f"updated:>={since_date}"
+    )
+    LOGGER.info(f"Searching PRs: {query}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)